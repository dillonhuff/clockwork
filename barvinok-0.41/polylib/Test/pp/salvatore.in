1 5
0 0  1  -1 0    x = a

1 3
0 0 0

a

