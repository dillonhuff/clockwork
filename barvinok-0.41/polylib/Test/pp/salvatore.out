---------------------------------------
Overall rays :
Line:   [   1    0  ]
---------------------------------------
Domain :
          1 >= 0

Vertices :
[ 0,  a ]
